wasm/event-stubs/Earthfile

```diff
@@ -0,0 +1,32 @@
+VERSION 0.7
+
+# cspell: words autodrop CFLAGS mexec
+
+# build-c-bindings - generates `wit-bindgen` with C, this should generate `hermes.c`, `hermes.h`, and `hermes_component_type.o`
+build-c-bindings:
+    FROM ../wasi-hermes-component-adapter+builder
+
+    # FIXME: right now we need to remove `--no-object-file` flag to make it runs with our bench, should we add this back?
+    RUN wit-bindgen c --autodrop-borrows yes ../wasi/wit
+
+    SAVE ARTIFACT ./
+
+# build-c-stub - compiles the generated C files into .wasm module
+build-c-stub:
+    FROM ghcr.io/webassembly/wasi-sdk:wasi-sdk-21
+    WORKDIR /repo
+    COPY . .
+    COPY +build-c-bindings/ ./
+
+    RUN $CC hermes.h hermes_component_type.o my-component.c -o out.wasm -mexec-model=reactor $CFLAGS
```

Comment:
Better place what exact compiler you are using right here instead of `$CC`.
Because it is possible that somehow `CC` env var would be changed and this line will not work anymore.

Reply:
And the same for the `$CFLAGS` better put flags explicitly